### ISSUE TITLE
saw client dashboard `running...` forever

### ISSUE THREAD

**Issue by weaversa** [GaloisInc/saw-script#1664]
There doesn't seem to be much documentation on the `myxine` dashboard for saw-remote-api. I saw a demo here: https://www.youtube.com/watch?v=8dBBeNSNRCQ and tried to run it.

The salsa20 example produces _almost_ the same output as the video above. Except, the final `successfully verified` message never shows up, it just keeps saying `running...`.

![image](https://user-images.githubusercontent.com/7407930/168504182-63dd3fd5-2ceb-488a-adc3-1a0fbe096367.png)

Is there some command I need to send back to the dashboard to let it know I've done all the unittests? We never get to see the bottom of @david-christiansen's  emacs buffer in the video, and the client syntax has changed quite a bit since then.

**Comment by jldodds** [comment]
This worked at the time of the demo. I'm not sure how actively we'll be able to continue developing this particular feature, since the library it's based on has also stopped development. Regardless, we should try to achieve this functionality. @robdockins has a script that uses the SAW summary JSON to create dot graphs. It's checked in and he can point us to where it is and how to use it.

**Comment by RyanGlScott** [comment]
Indeed, my understanding is that this demo was based off of functionality in a much older version of `argo` (which has since been split off into the SAW Python bindings) that no longer exists. We would need to investigate to see how to recover this functionality in terms of [`verif-viewer`](https://github.com/GaloisInc/saw-script/tree/c414366fe2634722076a2508b2773a63848919b3/verif-viewer) (which is the SAW summary JSON script mentioned in https://github.com/GaloisInc/saw-script/issues/1664#issuecomment-1129100350).
